### ISSUE TITLE
Need to do a phase sweep

### ISSUE THREAD

**Issue by gwenwhite** [cmelab/Advanced-Sampling#38]
Also need to calculate average PE, variance in PE, RDF, and average nearest neighbors. 
Additionally, we should think and chat about how we should do things (mixing, etc).

**Comment by chrisjonesBSU** [comment]
What are we thinking for things like:

1. Mixing step temperature and length
2. How many steps to run for in the sampling portion
3. Number of particles

I'll start a PR with a branch for this phase sweep to make sure we're all consistently using the same values for the parameters we aren't changing.

**Comment by chrisjonesBSU** [comment]
Here is what I'm thinking for the phase space:

```
# State points:
temps = np.arange(0.2, 5.2, 0.2)
densities = np.arange(0.1, 0.95, 0.05)

gwen_temps = temps[0:13]
mad_temps = temps[13:]
chris_temps = gwen_temps
marjan_temps = mad_temps

gwen_density = densities[0:8]
chris_density = densities[8:]
mad_density = chris_density
marjan_density = gwen_density
```

@gwenwhite  state points are:
```
N_Density: [0.1  0.15 0.2  0.25 0.3  0.35 0.4  0.45]
Temperatures: [0.2 0.4 0.6 0.8 1.  1.2 1.4 1.6 1.8 2.  2.2 2.4 2.6]
```

@marjanAlbouye state points are:
```
N_Density: [0.1  0.15 0.2  0.25 0.3  0.35 0.4  0.45]
Temperatures: [2.8 3.  3.2 3.4 3.6 3.8 4.  4.2 4.4 4.6 4.8 5. ]
```

@madilynpaul state points are:
```
N_Density: [0.5  0.55 0.6  0.65 0.7  0.75 0.8  0.85 0.9 ]
Temperatures: [2.8 3.  3.2 3.4 3.6 3.8 4.  4.2 4.4 4.6 4.8 5. ]
```

@chrisjonesBSU  state points are:
```
N_Density: [0.5  0.55 0.6  0.65 0.7  0.75 0.8  0.85 0.9 ]
Temperatures: [0.2 0.4 0.6 0.8 1.  1.2 1.4 1.6 1.8 2.  2.2 2.4 2.6]
```

**Comment by chrisjonesBSU** [comment]
You can check out the `phase-sweep` branch to see how I'm setting up the parameters we want to hold constant and the simulation run schedule in `project.py`

**Comment by gwenwhite** [comment]
Should the kT values have two numbers in them? Like this for example: `parameters["kT"] = [[10,0.2]]`

**Comment by chrisjonesBSU** [comment]
> Should the kT values have two numbers in them? Like this for example: `parameters["kT"] = [[10,0.2]]`

Yes, remember that we first want to mix the system up at a really high temperature (the first temperature), then run the system at the temperature we actually care about (the second temperature)

**Comment by gwenwhite** [comment]
Ok that sounds good. Thanks you!

**Comment by marjanAlbouye** [comment]
Thank you @chrisjonesBSU. Should we all set `hard_sphare` to `False`?

**Comment by chrisjonesBSU** [comment]
> Thank you @chrisjonesBSU. Should we all set `hard_sphare` to `False`?

I think so, I just changed it to `False`
